XenModel/Messages.resx

```diff
@@ -1834,6 +1834,27 @@ This alarm is set to be triggered when the total throughput exceeds {4}.</value>
   <data name="ALERT_CPUS_SUB_TEXT" xml:space="preserve">
     <value>When CPU usage exceeds {0}% for {1} min(s)</value>
   </data>
+  <data name="ALERT_CSS_EXPIRED_DESCIRPTION" xml:space="preserve">
+    <value>Your Customer Success Services (CSS) agreement expired on {0}.</value>
+  </data>
+  <data name="ALERT_CSS_EXPIRED_LINK_TEXT" xml:space="preserve">
+    <value>Renew CSS Agreement</value>

```

Comment:
should we then change it to the wording used in the license changes PR, or am I changing it there?

Reply:
I understood that in the case of expiry the license manager should also read `Renew CSS agreement`.